test/mlir/accelerators/nnpa/conversion/rewrite-onnx-for-zhigh.mlir

```diff
@@ -527,3 +527,12 @@ func.func @test_onnx_conv2d_not_insert_onnxpad_when_not_necessary(%arg0: tensor<
   // CHECK-LABEL: test_onnx_conv2d_not_insert_onnxpad_when_not_necessary
   // CHECK-NOT: "onnx.Pad"
 }
+
+// -----
+
+func.func @test_onnx_conv2d_not_insert_onnxpad_if_cannot_get_pads(%arg0: tensor<1x3x?x?xf32>, %arg1 : tensor<64x3x7x7xf32>, %arg2 : tensor<64xf32>) -> tensor<1x64x?x?xf32> {
+    %0 = "onnx.Conv"(%arg0, %arg1, %arg2) {auto_pad = "NOTSET", group = 1 : si64, kernel_shape = [7, 7], pads = [3, 3, 3, 3], strides = [2, 2]} : (tensor<1x3x?x?xf32>, tensor<64x3x7x7xf32>, tensor<64xf32>) -> tensor<1x64x?x?xf32>
+    return %0 : tensor<1x64x?x?xf32>
+  // CHECK-LABEL: test_onnx_conv2d_not_insert_onnxpad_if_cannot_get_pads
+  // CHECK-NOT: "onnx.Pad"
+}
```

Comment:
Could you add a lit test for the condition related to `VALID_PADDING`?

Reply:
Appended a lit test for the onnx.Conv with auto_pad==VALID case. Thank you.